File: clients/csharp/src/Swagger/Client/Model/Stats.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Swagger.Client.Model {
  public class Stats {
    public string rootSymbol { get; set; }

    public string currency { get; set; }

    public double? volume24h { get; set; }

    public double? turnover24h { get; set; }

    public double? openInterest { get; set; }

    public double? openValue { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Stats {\n");
      sb.Append("  rootSymbol: ").Append(rootSymbol).Append("\n");
      sb.Append("  currency: ").Append(currency).Append("\n");
      sb.Append("  volume24h: ").Append(volume24h).Append("\n");
      sb.Append("  turnover24h: ").Append(turnover24h).Append("\n");
      sb.Append("  openInterest: ").Append(openInterest).Append("\n");
      sb.Append("  openValue: ").Append(openValue).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: clients/android/src/main/java/io/swagger/client/model/Stats.java
package io.swagger.client.model;

import java.math.BigDecimal;

import io.swagger.annotations.*;
import com.google.gson.annotations.SerializedName;


@ApiModel(description = "")
public class Stats  {
  
  @SerializedName("rootSymbol")
  private String rootSymbol = null;
  @SerializedName("currency")
  private String currency = null;
  @SerializedName("volume24h")
  private BigDecimal volume24h = null;
  @SerializedName("turnover24h")
  private BigDecimal turnover24h = null;
  @SerializedName("openInterest")
  private BigDecimal openInterest = null;
  @SerializedName("openValue")
  private BigDecimal openValue = null;

  
  /**
   **/
  @ApiModelProperty(required = true, value = "")
  public String getRootSymbol() {
    return rootSymbol;
  }
  public void setRootSymbol(String rootSymbol) {
    this.rootSymbol = rootSymbol;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public String getCurrency() {
    return currency;
  }
  public void setCurrency(String currency) {
    this.currency = currency;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public BigDecimal getVolume24h() {
    return volume24h;
  }
  public void setVolume24h(BigDecimal volume24h) {
    this.volume24h = volume24h;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public BigDecimal getTurnover24h() {
    return turnover24h;
  }
  public void setTurnover24h(BigDecimal turnover24h) {
    this.turnover24h = turnover24h;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public BigDecimal getOpenInterest() {
    return openInterest;
  }
  public void setOpenInterest(BigDecimal openInterest) {
    this.openInterest = openInterest;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public BigDecimal getOpenValue() {
    return openValue;
  }
  public void setOpenValue(BigDecimal openValue) {
    this.openValue = openValue;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Stats {\n");
    
    sb.append("  rootSymbol: ").append(rootSymbol).append("\n");
    sb.append("  currency: ").append(currency).append("\n");
    sb.append("  volume24h: ").append(volume24h).append("\n");
    sb.append("  turnover24h: ").append(turnover24h).append("\n");
    sb.append("  openInterest: ").append(openInterest).append("\n");
    sb.append("  openValue: ").append(openValue).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
